bundles/org.openhab.core.io.transport.modbus/pom.xml

```
@@ -35,7 +35,7 @@
     <dependency>
       <groupId>net.wimpi</groupId>
       <artifactId>jamod</artifactId>
-      <version>[1.3.0.OH,1.3.1.OH]</version>
+      <version>[1.3.2.OH]</version>
```

It seems unnecessary to still have brackets for a single version.

```suggestion
      <version>1.3.2.OH</version>
```